### PR TITLE
Create TOBIT REGRESSIONS

### PR DESCRIPTION
**Pull request by duae-marriam**

### CODE REVIEW

**Review by DominikZabinski — COMMENTED** [review]
Please add what libraries are needed and how you load the data.

**Comment by duae-marriam** [comment]
Added the part where required libraries are loaded and data is fetched.
Let me know if the rest is fine here

File: TOBIT REGRESSIONS [hunk 1]
```diff
@@ -1,4 +1,17 @@
-# Tobit regression models with the specified conditions and variables as per the original analysis
+# Installing the libraries
+
+library(censReg)
+library(broom)
+library(dplyr)
+library(tidyr)
+
+# loading data
+df <- read_csv("D:/4 Sem/RR/data_DMDDG.csv")
```
Comment (DominikZabinski) [inline TOBIT REGRESSIONS]: Could you please take a look at [this](https://github.com/DominikZabinski/rr_group_project/blob/main/supplementary.Rmd#L96-L102) part of code and change your so we can avoid reading files from local system and having hard-coded file paths? :)

**Comment by duae-marriam** [comment]
Ohh, i see your point.
I am replacing that part of loading the data with this chunk

**Review by DominikZabinski — APPROVED** [review]

**Pull request merged**